backend/src/libs/packages/open-ai/open-ai.package.ts

```diff
@@ -0,0 +1,76 @@
+import { type HTTPService } from '../http/http.js';
+import { DEFAULT_IMAGE_GENERATE_REQUEST } from './libs/constants/constants.js';
+import {
+  type OpenAiImageGenerateRequestDto,
+  type OpenAiImageGenerateResponseDto,
+  type OpenAiMessageGenerateRequestDto,
+  type OpenAiMessageGenerateResponseDto,
+} from './libs/types/types.js';
+
+type OpenAiDependencies = {
+  httpService: HTTPService;
+  apiKey: string;
+  baseUrl: string;
+  model?: string;
+};
+
+class OpenAi {
+  private apiKey: string;
+  private httpService: HTTPService;
+  private baseUrl: string;
+  private model: string;
+
+  public constructor({
+    httpService,
+    apiKey,
+    baseUrl,
+    model = 'gpt-3.5-turbo-0301',
+  }: OpenAiDependencies) {
+    this.httpService = httpService;
+    this.apiKey = apiKey;
+    this.baseUrl = baseUrl;
+    this.model = model;
+  }
+
+  public async getMessageResponse({
+    content,
+  }: OpenAiMessageGenerateRequestDto): Promise<string | null> {
+    const data = await this.httpService.load<OpenAiMessageGenerateResponseDto>({
+      method: 'POST',
+      url: `${this.baseUrl}chat/completions`,
+      data: {
+        model: this.model,
+        messages: [
+          {
+            role: 'user',
+            content,
+          },
+        ],
+      },
+      token: this.apiKey,
+    });
+
+    const [response] = data.choices;
+
+    return response?.message.content ?? null;
```

Comment:
```suggestion
    return response?.message?.content ?? null;
```

Reply:
this won't work - it is an Unnecessary optional chain on a non-nullish value.

Reply:
If non-nullish then remove optional chaining